src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -34,14 +45,24 @@
 }
 
 // Group
-.pf-l-toolbar__group:not(:last-child) {
+.pf-l-toolbar__group {
+  // adds top margin to all group divs
+  margin-top: var(--pf-l-toolbar__group--MarginTop);
   margin-right: var(--pf-l-toolbar__group--MarginRight);
+
+  .pf-l-toolbar__item {
+    margin-top: 0;
+    margin-right: 0;
```

Comment:
I would make this and the line above it variables so all of this spacing is customizable. You can just set the vars to `0`.

Reply:
- added `--pf-l-toolbar__item--MarginRight-Zero: var(0);` variable to top list
- provided correct variables for `margin-top` and `margin-right` with new top list variable `--pf-l-toolbar__item--MarginRight-Zero`